README.md

```diff
@@ -6,13 +6,15 @@ sandbox for implementation of grid_map and navigation related code.
 
 ## Pre-requirements
 
+On Ubuntu:
+
 ```bash
 sudo apt-get install g++ pkg-config libx11-dev libasound2-dev libudev-dev protobuf-compiler
 ```
 
 ## Try examples
 
 ```bash
-cargo build --examples --release
-./target/release/dwa_gui
+cargo build --bins --examples --release
+./target/release/examples/dwa_gui
```

Comment:
It might be better to use `cargo run --example`.

Reply:
Updated to use that.